sandbox/tests/test scenes/bloom/_shaders/as_blender_areaLight.oso

```diff
@@ -0,0 +1,37 @@
+OpenShadingLanguage 1.00
```

Comment:
These shaders are already in the sandbox I think. Can you try to reference them instead of duplicating them ? You can try `grep -rn as_blender` in the `test scenes` folder to see how other scenes are using it.